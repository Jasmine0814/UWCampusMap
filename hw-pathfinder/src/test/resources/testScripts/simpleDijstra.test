# Find a path with Dijkstra's algorithm in simple graph

CreateGraph Graph1

AddNode Graph1 A
AddNode Graph1 B

AddEdge Graph1 A B 1.1
AddEdge Graph1 B A 2.2

FindPath Graph1 A B





