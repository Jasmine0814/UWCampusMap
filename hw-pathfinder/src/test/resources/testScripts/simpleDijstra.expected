# Find a path with Dijkstra's algorithm in simple graph

created graph Graph1

added node A to Graph1
added node B to Graph1

added edge 1.100 from A to B in Graph1
added edge 2.200 from B to A in Graph1

path from A to B:
A to B with weight 1.100
total cost: 1.100
